project/cmake/scripts/linux/ExtraTargets.cmake

```diff
@@ -7,6 +7,9 @@ if(ENABLE_X11 AND X_FOUND AND XRANDR_FOUND)
 endif()
 
 # WiiRemote
-if(ENABLE_EVENTCLIENTS AND BLUETOOTH_FOUND AND CWIID_FOUND)
-  add_subdirectory(${CORE_SOURCE_DIR}/tools/EventClients/Clients/WiiRemote build/WiiRemote)
+if(ENABLE_EVENTCLIENTS AND BLUETOOTH_FOUND)
+  find_package(CWiid QUIET)
+  if(CWIID_FOUND)
+    add_subdirectory(${CMAKE_SOURCE_DIR}/tools/EventClients/Clients/WiiRemote build/WiiRemote)
```

Comment:
CORE_SOURCE_DIR should be kept, otherwise configure fails:

-- Found CWIID: /home/buildroot/br8_ffmpeg3_kodi17_github/output/host/usr/x86_64-buildroot-linux-uclibc/sysroot/usr/lib64/libcwiid.so (found version "0.6.00") 
CMake Error at scripts/linux/ExtraTargets.cmake:13 (add_subdirectory):
  add_subdirectory given source
  "/home/buildroot/br8_ffmpeg3_kodi17_github/output/build/kodi-07daa5578e1d20eb056ce064baa42dbd47c86be2/project/cmake/tools/EventClients/Clients/WiiRemote"
  which is not an existing directory.